mini-kodein-android-compose/mini-kodein-android.pro

```diff
@@ -0,0 +1,2 @@
+# Needed for injection
```

Comment:
BTW, rename this file to `mono-kodein-android-compose.pro`

Reply:
63e044a01e8f169b2add9ea95905f36e99de4207